src/test/java/gameframework/game/GameDataTest.java

```diff
@@ -0,0 +1,79 @@
+package gameframework.game;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import gameframework.base.ObservableValue;
+
+public class GameDataTest {
+
+	GameData gd;
+	
+	@Before
+	public void init()	{
+		// GameConfiguration is 100 test-covered, thus trusted
+		gd = new GameData(new GameConfiguration());
+	}
+	
+	@Test
+	public void GameDataConstructorTest()	{
+		assertNotNull(gd.getCanvas());
+		
+		// Score must be equals to 0 (default values)
+		assertNotNull(gd.getScore());
+		assertEquals(new ObservableValue<Integer>(0).getValue(), gd.getScore().getValue());
+		
+		// Life must be equals to 2 (default values)
+		assertNotNull(gd.getLife());
+		assertEquals(new ObservableValue<Integer>(2).getValue(), gd.getLife().getValue());
+		
+		// EndOfGame boolean should be false (default values)
+		assertNotNull(gd.getEndOfGame());
+		assertEquals(new ObservableValue<Boolean>(false).getValue(), gd.getEndOfGame().getValue());
+		
+		// And Levels map should be empty but not null
+		assertNotNull(gd.getLevels());
+		assertEquals(0, gd.getLevels().size());
+		
+		// Everything else is up to GameConfiguration
+	}
+	
+	
+	@Test
+	public void increaseLifeTest()	{
+		gd.increaseLife(25);
```

Comment:
please add an `assert` with value `2` before so it's clear that the previous value is 2.